packages/react-native/src/View.tsx

```diff
@@ -169,14 +175,17 @@ export class View {
 
     managerAddons.loadAddons({
       store: () => ({
-        fromId: (id) =>
-          this._preview.storyStore.getStoryContext(this._preview.storyStore.fromId(id)),
+        fromId: async (id) => {
+          // return this._preview.storyStore.getStoryContext(this._preview.storyStore.fromId(id));
+
+          // console.log()
+          // this._preview.store
+          // console.log('fromId', id);
+          return this._preview.loadStory({ storyId: id });
```

Comment:
@shilman @tmeasday this loadStory is async and makes it hard to work with. Also its just not returning the story for some reason. Are we missing something?

Reply:
```
return this._preview.getStoryContext(this._idToPrepared[id]);
```